From 396ec1962a2678d50b9668b48b8a74784cb9aced Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:57 -0400
Subject: [redhat] kabi: Adding symbol kthread_create_on_node (kernel/kthread.c)

Message-id: <6f9dcc7c727a9126690d2d6c5f66e9cc19586813.1410510773.git.poros@redhat.com>
Patchwork-id: 95960
O-Subject: [PATCH RHEL7.1 117/233] kabi: Adding symbol kthread_create_on_node (kernel/kthread.c)
Bugzilla: 1062361
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Not used by PowerPath directly. The Macro kthread_run invokes
    the macro kthread_create; the kthread_macro calls this function

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/kthread_create_on_node
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/kthread_create_on_node
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/kthread_create_on_node

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/kthread_create_on_node b/redhat/kabi/kabi-module/kabi_ppc64/kthread_create_on_node
new file mode 100644
index 0000000..52e8af3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/kthread_create_on_node
@@ -0,0 +1,2 @@
+#1-
+0x7238a29a kthread_create_on_node vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/kthread_create_on_node b/redhat/kabi/kabi-module/kabi_s390x/kthread_create_on_node
new file mode 100644
index 0000000..f8efc52
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/kthread_create_on_node
@@ -0,0 +1,2 @@
+#1-
+0xb7488438 kthread_create_on_node vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/kthread_create_on_node b/redhat/kabi/kabi-module/kabi_x86_64/kthread_create_on_node
new file mode 100644
index 0000000..c4fa6f7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/kthread_create_on_node
@@ -0,0 +1,2 @@
+#1-
+0xc35e4b4e kthread_create_on_node vmlinux EXPORT_SYMBOL
-- 
1.7.1